CHANGELOG.md

```diff
@@ -9,6 +9,17 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 ## Latest
+* Use hint diagnostics in vscode for better rendering in the editor (PR: [#69403](https://github.com/dotnet/roslyn/pull/69403))
```

Comment:
```suggestion
* Change how hint diagnostics are shown so they aren't shown as blue squiggles (PR: [#69403](https://github.com/dotnet/roslyn/pull/69403))
```

Reply:
(that way folks understand the "blue squiggle" problem is addressed and they can potentially revert workarounds)